tuxemon/graphics.py

```diff
@@ -271,7 +271,7 @@ def animation_frame_files(
 
     """
     frames = list()
-    pattern = re.compile(rf"{name}\.[0-9]+\.png")
+    pattern = re.compile(rf"{name}\.?[0-9]+\.png")
```

Comment:
If I understand correctly, we allow animations to not have the `xx` frame index?

The animation download script will always generate the frame index, so if only 1 frame is present, it will be `animation_name00.png`.

We could also force the regex to take only 2 digs in the frame index, unless it's okay for an animation to have more than 100 frames.

Reply:
The previous regex supported

foo.000.png

It now supports that and:

foo00.png

I just made the first dot optional.